x-pack/plugins/security_solution/server/endpoint/services/actions/clients/sentinelone/types.ts

```diff
@@ -0,0 +1,26 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+/* eslint-disable @typescript-eslint/no-explicit-any */
+
+import type { ActionsClient } from '@kbn/actions-plugin/server';
+import type { SUB_ACTION } from '@kbn/stack-connectors-plugin/common/sentinelone/constants';
+
+type ConnectorActionsExecuteOptions = Parameters<ActionsClient['execute']>[0];
```

Comment:
```suggestion
type ConnectorActionsExecuteOptions = Parameters<ActionsClient['execute']>[number];
```

Reply:
no... in this case, I want the first argument of the `execute()` method - not any of the arguments. So if I expand the signature of the method to had additional arguments (not just the one), this type will remain compatible here.

Reply:
Ah I see! That makes sense.